docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1g.md

```diff
@@ -0,0 +1,45 @@
+# does_zone_meet_G3_1_1g
+**Schema Version:** 0.0.22  
+
+**Description:** returns a boolean value indicating whether the zone meets G3_1_1g:
+ 
+**part1:** "If the baseline HVAC system serves HVAC zones that includes computer rooms,  baseline system 11 shall be used where the baseline HVAC system type is 7 or 8 and the total computer room peak cooling load is greater than 600,000 BTU/h (175 kW) AND <= 3,000,000"
+
+**part 2:** If the baseline HVAC system serves HVAC zones that includes computer rooms,  Baseline System 11 shall be used for such HVAC zones in buildings with a total computer room peak cooling load greater than 3,000,000 Btu/h.
+
+**part 3:** If the baseline HVAC system serves HVAC zones that includes computer rooms,  baseline system 3 or 4 shall be used for all HVAC other zones - that is zones where the computer room peak cooling load is= <600,000 Btu/h, and when the peak cooling load is between 600,000 btu/hr and 3,000,000 btu/hr and the baseline system type is not system 7 or 8.
+
+**Inputs:** 
+- **B_RMI**
+- **zone_id**
+- **starting_system_type**
+
+**Returns:**  
+- **result**: an enum - either NO or G1, G2, G3 for the 3 parts
+
+**Function Call:**
+- **get_component_by_id**
+- **get_zones_computer_rooms**
+- **get_zone_peak_internal_load**
+
+## Logic:
+- set the result variable to NO - only a positive test can give it a different value: `result = NO`
+- get the list of zones with computer rooms using the function get_zones_computer_rooms to get a list of zones including computer rooms: `computer_room_zones_dict = get_zones_computer_rooms(B_RMI)`
+- if the zone_id is in the computer_room_zones_dict, then this is a computer room zone and it is eligible: `if zone_id in computer_room_zones_dict:`
+    - create a variable for the total peak computer room load: `total_computer_peak_cooling_load = 0`
+    - loop through the computer room zones: `for computer_room_zone_id in computer_room_zones_dict:`
+        - get the zone: `computer_room_zone = get_component_by_id(computer_room_zone_id)`
+        - get the peak internal load for the computer_room_zone by calling the function get_zone_peak_internal_load (units are btu/sf/hr): `zone_peak_internal_load = get_zone_peak_internal_load_floor_area(B_RMI, computer_room_zone)["PEAK"]`
```

Comment:
question: the app G requirement applies to each baseline HVAC system, and not all computer room zones in a B RMI.
so instead off looping through B RMI, should this be looking through each HVAC system in B RMI and then calculating peak cooling load for all computer room zones served by the baseline HVAC system?

Reply:
#2 has been updated.  I think we should discuss #1 on the call

Reply:
My interpretation for #1 is that it is based on the baseline cooling load, not proposed.